main.cpp

```diff
@@ -68,13 +68,13 @@ fist.bump();
 //  action 3:   pounds
 fist.pound();
 //  3)
-//  Noun:   legs
-//  action 1:   run
-legs.run();
-//  action 2:   jump
-legs.jump();
-//  action 3:   kick
-legs.kick();
+//  Noun:   leftFoot
+//  action 1:   leftRun
+leftFoot.leftRun();
+//  action 2:   leftJump
+leftFoot.leftJump();
+//  action 3:   leftKick
+leftFoot.leftKick();
 //  4)

```

Comment:
Please choose a different noun, and come up with 3 actions. 
`leftRun` `leftJump` `leftKick` are not good choices. 
Remember:
**Name things what they are or what they do.** 
Here's an example:
`fordMustang.turnLeft();`
It satisfies the requirements:
- choose a noun that must be spelled using camel case
- choose an action that must be spelled using camel case.
- 
- [ ] revise